WTWSMS/common/religions/00_religions.txt

```diff
@@ -4378,14 +4439,14 @@ pagan_group = {
 		heresy_icon = 85
 		
 		color = { 0.5 0.05 0.0 }
-		#parent = hellenic_pagan
+		parent = hellenic_pagan

```

Comment:
What circumstances? That event is used for all heresies. If it doesn't fit for Mastimanis, we need to ask the question on whether or not they are Hellenic.